Makefile

```diff
@@ -475,7 +475,7 @@ restart-client: | stop-client run-client ## Restarts the webapp.
 
 run-job-server: ## Runs the background job server.
 	@echo Running job server for development
-	$(GO) run $(GOFLAGS) -ldflags '$(LDFLAGS)' $(PLATFORM_FILES) jobserver --disableconfigwatch &
+	$(GO) run $(GOFLAGS) -ldflags '$(LDFLAGS)' $(PLATFORM_FILES) jobserver --disableconfigwatch
```

Comment:
Why is this needed? Can affect the server workflow in some way?

Reply:
This made testing a lot easier (ctrl + c to stop instead of trying to kill the process by pid) but I'll add it back for master.